tsconfig.json

```
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "experimentalDecorators": true,
+        "emitDecoratorMetadata": true,
+        "outDir": "dist",
+        "sourceMap": true,
+        "strictNullChecks": true,
+        "declaration": true,
+        "noFallthroughCasesInSwitch": true,
+        "noImplicitReturns": true,
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "noUnusedParameters": true,
+        "pretty": true,
+        "strict": true,
+        "downlevelIteration": true,
+        "target": "es5",
+        "lib": [
+            "es2015"
+        ]
+    },
+    "include": [
+        "src/**/*",
+    ],
```

Trailing comma is not allowed in JSON.

```suggestion
    ]
```